packages/core/src/components/context-menu/_context-menu.scss

```diff
@@ -110,6 +110,10 @@ rightClickMe.oncontextmenu = (e: MouseEvent) => {
 Styleguide components.context-menu.js
 */
 
+.pt-context-menu .pt-popover-target {
+  display: block;
+}
+
 .pt-context-menu-popover-target {
   position: fixed;
 }
```

Comment:
What exactly is the difference between these two class types `.pt-context-menu .pt-popover-target` vs `.pt-context-menu-popover-target`?